index/manual_plantsvszombies2gardendless_trikehard.toml

```diff
@@ -0,0 +1,8 @@
+name = "manual_plantsvszombies2gardendless_trikehard"
+display_name = "Manual: Plants Vs Zombies 2"
+
+home = "https://discord.com/channels/1097532591650910289/1435957999809335368"
+
+[versions]
+# https://discord.com/channels/1097532591650910289/1435957999809335368
+"0.0.1" = { local = "../apworlds/manual_plantsvszombies2gardendless_trikehard" }
```

Comment:
Please add the version in the filename. Also it needs to match (you're missing the extension)